marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping-functions/getMappingFunctions.sjs

```diff
@@ -0,0 +1,11 @@
+const mappingFunctions = require('/data-hub/5/extensions/mappingFunctions.sjs');
+const test = require("/test/test-helper.xqy");
+
+function testMappingFunctions() {
+return [
+    test.assertEqual(129, Object.keys(mappingFunctions.getXpathFunctions()).length),
```

Comment:
I've done assertions like this before on other projects, but they tend to be brittle as this number can go up at any point. I recommend changing this to "assertTrue" that the number of functions is at least 129, and include an assertion message stating that as of ML version X.X, we expect at least 129 functions in the "fn:" namespace.